agir/events/components/EventSettings/EventSettings.js

```diff
@@ -43,6 +43,16 @@ export const EventSettings = (props) => {
   return (
     <ObjectManagement
       title={event?.name}
+      warning={
+        !event.isEditable ? (
+          <>
+            Cet événement n'est pas modifiable directement via Action Populaire.
+            <br />
+            Pour toute question, veuillez contacter nos équipes à l'adresse
+            e-mail <strong>groupes@actionpopulaire.fr</strong>
+          </>
+        ) : null
```

Comment:
Comme il n'y a pas de `else` et que le cast en booléen est déjà fait ? Mais cela ne me dérange pas plus si on garde les 2 manières :)
```suggestion
        !event.isEditable && <>
            Cet événement n'est pas modifiable directement via Action Populaire.
            <br />
            Pour toute question, veuillez contacter nos équipes à l'adresse
            e-mail <strong>groupes@actionpopulaire.fr</strong>
          </>
```

Reply:
C'est pas exactement le même cas ici, parce qu'il s'agit d'une prop du composant. Mais pour éviter les confusions je vais plutôt mettre `undefined` comme fallback.